find: './fixtures/frontend/ccvd.fixture': No such file or directory
find: './fixtures/frontend/pipe_out.do_match': No such file or directory
find: './fixtures/frontend/pipe_out.iterate_one': No such file or directory
find: './fixtures/frontend/vofrontend/signatures.sha1': No such file or directory
find: './fixtures/frontend/web-area/stage/description.e98f4o.cfg': No such file or directory
find: './fixtures/frontend/web-area/stage/file_list.j4j7WK.lst': No such file or directory
find: './fixtures/frontend/web-area/stage/group_group1/description.e98f4o.cfg': No such file or directory
find: './fixtures/frontend/web-area/stage/group_group1/nodes.blacklist': No such file or directory
find: './fixtures/frontend/web-area/stage/group_group1/signature.j4j84x.sha1': No such file or directory
find: './fixtures/frontend/web-area/stage/signature.j4j84x.sha1': No such file or directory
find: './test_creation_lib_cvWCreate.py': No such file or directory
find: './test_frontend_glideinFrontendConfig.py': No such file or directory
find: './test_frontend_glideinFrontendElement.py': No such file or directory
TESTING ==========> ./test_cleanupSupport.py

Running tests...
----------------------------------------------------------------------
Removed 10 files for 0.00MB.
.
----------------------------------------------------------------------
Ran 1 test in 0.015s

OK

Generating XML reports...
TESTING ==========> ./test_condorExe.py

Running tests...
----------------------------------------------------------------------
...
----------------------------------------------------------------------
Ran 3 tests in 5.574s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cWDictFile_DictFile.py

Running tests...
----------------------------------------------------------------------
....................
----------------------------------------------------------------------
Ran 20 tests in 0.028s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cWDictFile_DictFileTwoKeys.py

Running tests...
----------------------------------------------------------------------
.....................
----------------------------------------------------------------------
Ran 21 tests in 0.027s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cWDictFile_DictFileTwoKeys_SubClasses.py

Running tests...
----------------------------------------------------------------------
.....
----------------------------------------------------------------------
Ran 5 tests in 0.002s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cWDictFile_SHADicts.py

Running tests...
----------------------------------------------------------------------
.........
----------------------------------------------------------------------
Ran 9 tests in 0.006s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cWDictFile_validate_node.py

Running tests...
----------------------------------------------------------------------
...
----------------------------------------------------------------------
Ran 3 tests in 0.043s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cgWConsts.py

Running tests...
----------------------------------------------------------------------
.........
----------------------------------------------------------------------
Ran 9 tests in 0.002s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cgWDictFile.py

Running tests...
----------------------------------------------------------------------
SSS.....SSS...SSSSSSSSSSSS..SSSS.S....S.............S.......SSS...
----------------------------------------------------------------------
Ran 66 tests in 0.955s

OK (skipped=28)

Generating XML reports...
TESTING ==========> ./test_creation_lib_cgWParamDict.py

Running tests...
----------------------------------------------------------------------
SSSSSS.......SSSSSSSS
----------------------------------------------------------------------
Ran 21 tests in 2.199s

OK (skipped=14)

Generating XML reports...
TESTING ==========> ./test_creation_lib_cgWParams.py

Running tests...
----------------------------------------------------------------------
..........S
----------------------------------------------------------------------
Ran 11 tests in 0.400s

OK (skipped=1)

Generating XML reports...
TESTING ==========> ./test_creation_lib_cvWConsts.py

Running tests...
----------------------------------------------------------------------
.......
----------------------------------------------------------------------
Ran 7 tests in 0.002s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cvWParamDict.py
/test_dir/venv-2.7/lib/python2.7/site-packages/htcondor/__init__.py:25: UserWarning: 
Using a null condor_config.
Neither the environment variable CONDOR_CONFIG, /etc/condor/,
/usr/local/etc/, nor ~/condor/ contain a condor_config source.
  warnings.warn(message)

Running tests...
----------------------------------------------------------------------
SS..S.SSS....S.SSSSS
----------------------------------------------------------------------
Ran 20 tests in 0.414s

OK (skipped=12)

Generating XML reports...
TESTING ==========> ./test_creation_lib_cvWParams.py

Running tests...
----------------------------------------------------------------------
...................
----------------------------------------------------------------------
Ran 19 tests in 0.652s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_factoryXmlConfig.py

Running tests...
----------------------------------------------------------------------
...........................
----------------------------------------------------------------------
Ran 27 tests in 1.439s

OK

Generating XML reports...
TESTING ==========> ./test_encodingSupport.py

Running tests...
----------------------------------------------------------------------
.....
----------------------------------------------------------------------
Ran 5 tests in 0.003s

OK

Generating XML reports...
TESTING ==========> ./test_factory_glideFactoryConfig.py

Running tests...
----------------------------------------------------------------------
.......................................................................................+++
.........................................................................................+++
...
----------------------------------------------------------------------
Ran 14 tests in 1.265s

OK

Generating XML reports...
TESTING ==========> ./test_factory_glideFactoryEntry.py
/test_dir/venv-2.7/lib/python2.7/site-packages/htcondor/__init__.py:25: UserWarning: 
Using a null condor_config.
Neither the environment variable CONDOR_CONFIG, /etc/condor/,
/usr/local/etc/, nor ~/condor/ contain a condor_config source.
  warnings.warn(message)

Running tests...
----------------------------------------------------------------------
S.SS.......SS...SSS.S.SS.SSS.S.S.SSSSSSS
----------------------------------------------------------------------
Ran 40 tests in 2.971s

OK (skipped=23)

Generating XML reports...
TESTING ==========> ./test_factory_glideFactoryLib.py
/test_dir/venv-2.7/lib/python2.7/site-packages/htcondor/__init__.py:25: UserWarning: 
Using a null condor_config.
Neither the environment variable CONDOR_CONFIG, /etc/condor/,
/usr/local/etc/, nor ~/condor/ contain a condor_config source.
  warnings.warn(message)

Running tests...
----------------------------------------------------------------------
SSS.SSSSSSSSSSSSSSSS.........S.........SSSSSSSSSSSSSSS.SSSSSSSSSSSS.SS.
----------------------------------------------------------------------
Ran 71 tests in 0.019s

OK (skipped=49)

Generating XML reports...
TESTING ==========> ./test_frontend.py
/test_dir/venv-2.7/lib/python2.7/site-packages/htcondor/__init__.py:25: UserWarning: 
Using a null condor_config.
Neither the environment variable CONDOR_CONFIG, /etc/condor/,
/usr/local/etc/, nor ~/condor/ contain a condor_config source.
  warnings.warn(message)

Running tests...
----------------------------------------------------------------------
.........................Running glidein ids at ('submit.local', 'Site_Name2@v3_0@factory1', 'frontend@factory1') (total glideins: 3, total jobs 4, cluster matches: 4): glidein_5@cmswn005.local, glidein_2@cmswn002.local, glidein_1@cmswn006.local
Running glidein ids at ('submit.local', 'Site_Name1@v3_0@factory1', 'frontend@factory1') (total glideins: 1, total jobs 1, cluster matches: 1): glidein_1@cmswn001.local
Running glidein ids at ('submit.local', 'Site_Name3@v3_0@factory1', 'frontend@factory1') (total glideins: 0, total jobs 0, cluster matches: 0): 
Running glidein ids at ('submit.local', 'Site_Name2@v3_0@factory1', 'frontend@factory1') (total glideins: 0, total jobs 0, cluster matches: 0): 
Running glidein ids at ('submit.local', 'Site_Name1@v3_0@factory1', 'frontend@factory1') (total glideins: 0, total jobs 0, cluster matches: 0): 
Running glidein ids at ('submit.local', 'Site_Name3@v3_0@factory1', 'frontend@factory1') (total glideins: 0, total jobs 0, cluster matches: 0): 
Running glidein ids at ('submit.local', 'Site_Name2@v3_0@factory1', 'frontend@factory1') (total glideins: 1, total jobs 1, cluster matches: 1): glidein_2@cmswn002.local
Running glidein ids at ('submit.local', 'Site_Name1@v3_0@factory1', 'frontend@factory1') (total glideins: 1, total jobs 1, cluster matches: 1): glidein_1@cmswn001.local
Running glidein ids at ('submit.local', 'Site_Name3@v3_0@factory1', 'frontend@factory1') (total glideins: 0, total jobs 0, cluster matches: 0): 
........
----------------------------------------------------------------------
Ran 33 tests in 1.606s

OK

Generating XML reports...
TESTING ==========> ./test_frontend_element.py
/test_dir/venv-2.7/lib/python2.7/site-packages/htcondor/__init__.py:25: UserWarning: 
Using a null condor_config.
Neither the environment variable CONDOR_CONFIG, /etc/condor/,
/usr/local/etc/, nor ~/condor/ contain a condor_config source.
  warnings.warn(message)

Running tests...
----------------------------------------------------------------------
..
----------------------------------------------------------------------
Ran 2 tests in 0.037s

OK

Generating XML reports...
TESTING ==========> ./test_glideFactoryDowntimeLib.py
/test_dir/venv-2.7/lib/python2.7/site-packages/htcondor/__init__.py:25: UserWarning: 
Using a null condor_config.
Neither the environment variable CONDOR_CONFIG, /etc/condor/,
/usr/local/etc/, nor ~/condor/ contain a condor_config source.
  warnings.warn(message)

Running tests...
----------------------------------------------------------------------
.....
----------------------------------------------------------------------
Ran 5 tests in 0.037s

OK

Generating XML reports...
TESTING ==========> ./test_glideinFrontendPlugins.py
/test_dir/venv-2.7/lib/python2.7/site-packages/htcondor/__init__.py:25: UserWarning: 
Using a null condor_config.
Neither the environment variable CONDOR_CONFIG, /etc/condor/,
/usr/local/etc/, nor ~/condor/ contain a condor_config source.
  warnings.warn(message)

Running tests...
----------------------------------------------------------------------
.....
----------------------------------------------------------------------
Ran 5 tests in 0.069s

OK

Generating XML reports...
TESTING ==========> ./test_lib_encodingSupport.py

Running tests...
----------------------------------------------------------------------
....
----------------------------------------------------------------------
Ran 4 tests in 0.834s

OK

Generating XML reports...
TESTING ==========> ./test_lib_exprParser.py

Running tests...
----------------------------------------------------------------------
...
----------------------------------------------------------------------
Ran 3 tests in 0.072s

OK

Generating XML reports...
TESTING ==========> ./test_lib_fork.py

Running tests...
----------------------------------------------------------------------
.............
----------------------------------------------------------------------
Ran 13 tests in 72.103s

OK

Generating XML reports...
TESTING ==========> ./test_lib_pubCrypto.py

Running tests...
----------------------------------------------------------------------
...++++++
........................................................................++++++
...............++++++
....................++++++
...............++++++
.........++++++
.
----------------------------------------------------------------------
Ran 3 tests in 0.114s

OK

Generating XML reports...
TESTING ==========> ./test_lib_servicePerformance.py

Running tests...
----------------------------------------------------------------------
..........
----------------------------------------------------------------------
Ran 10 tests in 0.003s

OK

Generating XML reports...
TESTING ==========> ./test_lib_symCrypto.py

Running tests...
----------------------------------------------------------------------
.......S........
----------------------------------------------------------------------
Ran 16 tests in 1.793s

OK (skipped=1)

Generating XML reports...
TESTING ==========> ./test_lib_timeConversion.py

Running tests...
----------------------------------------------------------------------
................
----------------------------------------------------------------------
Ran 16 tests in 0.428s

OK

Generating XML reports...
TESTING ==========> ./test_lib_x509Support.py

Running tests...
----------------------------------------------------------------------
.
----------------------------------------------------------------------
Ran 1 test in 0.030s

OK

Generating XML reports...
TESTING ==========> ./test_lib_xmlParse.py

Running tests...
----------------------------------------------------------------------
.................
----------------------------------------------------------------------
Ran 17 tests in 0.017s

OK

Generating XML reports...
TESTING ==========> ./test_logSupport.py

Running tests...
----------------------------------------------------------------------
./test_logSupport.py:35: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.config = yaml.load(file(config_file, 'r'))
....
----------------------------------------------------------------------
Ran 4 tests in 184.544s

OK

Generating XML reports...
TESTING ==========> ./test_renew_proxy.py

Running tests...
----------------------------------------------------------------------
...
----------------------------------------------------------------------
Ran 3 tests in 0.001s

OK

Generating XML reports...
TESTING ==========> ./test_tarSupport.py

Running tests...
----------------------------------------------------------------------
..
----------------------------------------------------------------------
Ran 2 tests in 0.020s

OK

Generating XML reports...
TESTING ==========> ./test_util.py

Running tests...
----------------------------------------------------------------------
.
----------------------------------------------------------------------
Ran 1 test in 0.000s

OK

Generating XML reports...
No source for code: '/test_dir/glideinwms/unittests/test_creation_lib_cvWCreate.py'.
Aborting report output, consider using -i.
coverage html report failed
